auditlog_security/views/auditlog_view.xml

```diff
@@ -0,0 +1,165 @@
+        <field name="model">auditlog.log.line</field>
+        <field name="arch" type="xml">
+                <form string="Log - Field updated" delete="0">
+                    <group>
+                        <field name="field_id" readonly="1" />
+                    </group>
+                    <group string="Values" col="4">
+                        <field name="create_date" readonly="1" />
+                        <field name="user_id" readonly="1" />
+                        <field name="old_value" readonly="1" />
+                        <field name="new_value" readonly="1" />
+                        <field name="old_value_text" readonly="1" />
+                        <field name="new_value_text" readonly="1" />
+                    </group>
+                </form>
+        </field>
+    </record>
+
+    <record model="ir.ui.view" id="view_auditlog_log_line_tree">
+        <field name="name">auditlog.log.line.tree</field>
+        <field name="model">auditlog.log.line</field>
+        <field name="arch" type="xml">
+            <tree delete="0">
+                <field name="create_date" />
+                <field name="field_name" string="Changed" />
+                <field name="old_value_text" />
+                <field name="new_value_text" />
+                <field name="user_id" />
+                <field name="method" string="Type of operation" />
+            </tree>
+        </field>
+    </record>
+
+    <record id="audit_log_line_action" model="ir.actions.act_window">
+        <field name="name">View Log Lines</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">auditlog.log.line</field>
+        <field name="view_mode">tree,form</field>
+    </record>
+
+    <!-- show logs for partner out side of technical too, now that we have rules
+    for all types of users, will place in "contact" -->
+
+
+    <record id="auditlog_form_view_ext" model="ir.ui.view">
+        <field name="name">auditlog rule form extension</field>
+        <field name="model">auditlog.rule</field>
+        <field name="inherit_id" ref="auditlog.view_auditlog_rule_form" />
+        <field name="arch" type="xml">
+            <xpath expr="//group[last()]" position="after">
+                <group>
+                    <div>
+                        <p>
+                            Add fields here to make any changes to them (audit log lines)
+                            visible to members of the selected groups.
+                        </p>
+                    </div>
+                </group>
+                <group>
+                    <field name="log_selected_fields_only" />
+                    <!-- in the domain are the blacklisted fields in auditlog they
+                        would be skipped anyway, so we repeat here to avoid incongruencies
+                    -->
+                    <field name="auditlog_line_access_rule_ids">
+                        <tree>
```

Comment:
So records created here are truly created, but become invisible as soon as we save the form view of the parent record. Here we are creating a one2many line for a model that holds two many2many fields.   Wondering if this is the reason

Reply:
... and this one because of a record rule. So basically no one can read these lines. Why?